### PR TITLE
[INDEX] feat: index carbon  angular

### PR DESCRIPTION
**Pull request by andreancardona**
https://github.com/carbon-design-system/carbon-platform/issues/514

## Description:

- This PR indexes `carbon-components-angular`

## Note:

We still need the following thumbnails:

- [x] tooltip-icon
- [x] icon
- [x] input
- [x] list
- [x] radio
- [x] table
- [x] time-picker-select
- [x] context-menu

### CODE REVIEW

File: src/carbon.yml [hunk 1]
```diff
@@ -0,0 +1,433 @@
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@v1/carbon-resources.schema.json
+library:
+  id: carbon-components-angular
+  name: Carbon Components Angular
+  description: Angular implementation of Carbon Components.
+  externalDocsUrl: https://angular.carbondesignsystem.com/?path=/story/components-welcome--to-carbon-angular
```
Comment (francinelucca) [inline src/carbon.yml:6]:
```suggestion
  externalDocsUrl: https://angular.carbondesignsystem.com
```

File: src/carbon.yml [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+      - input-control
+  link:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/components/inline-loading/usage
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-link
```
Comment (francinelucca) [inline src/carbon.yml:172]: incorrect externalDocsUrl

File: src/carbon.yml [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+  #TODO: update - inline?
+  loading:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/components/inline-loading/usage
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-link
```
Comment (francinelucca) [inline src/carbon.yml:198]: icorrect externalDocsUrl, incorrect demoLink url

File: src/carbon.yml [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+  #TODO: update
+  number:
+      status: stable
+      framework: angular
+      externalDocsUrl: https://www.carbondesignsystem.com/components/notification/usage
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://angular.carbondesignsystem.com/?path=/story/components-number
```
Comment (francinelucca) [inline src/carbon.yml:226]: incorrect ExternalDocsUrl

File: src/carbon.yml [hunk 5]
```diff
@@ -0,0 +1,433 @@
...
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-input
+    #TODO: @aubrey check tag
+    tags:
+      - input-control
+  link:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/components/inline-loading/usage
```
Comment (andreancardona) [inline src/carbon.yml:167]:
```suggestion
    externalDocsUrl: https://carbondesignsystem.com/components/link/usage
```

File: src/carbon.yml [hunk 6]
```diff
@@ -0,0 +1,433 @@
...
+  grid:
+    status: stable
+    framework: angular
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-grid
+  icon:
+    name: icon
```
Comment (francinelucca) [inline src/carbon.yml:123]:
```suggestion
    name: Icon
```

File: src/carbon.yml [hunk 7]
```diff
@@ -0,0 +1,433 @@
...
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-list
+    #TODO: @aubrey check tag
+    tags:
+      - data-display
+  #TODO: update - inline?
+  loading:
+    status: stable
+    framework: angular
+    externalDocsUrl: https://www.carbondesignsystem.com/components/inline-loading/usage
```
Comment (andreancardona) [inline src/carbon.yml:193]:
```suggestion
    externalDocsUrl: https://carbondesignsystem.com/components/link/usage

File: src/carbon.yml [hunk 8]
```diff
@@ -0,0 +1,433 @@
...
```
Comment (andreancardona) [inline src/carbon.yml]:
```suggestion
    externalDocsUrl: https://carbondesignsystem.com/components/loading/usage/
```

File: src/carbon.yml [hunk 9]
```diff
@@ -0,0 +1,433 @@
...
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-link
```
Comment (andreancardona) [inline src/carbon.yml:198]:
```suggestion
        url: https://angular.carbondesignsystem.com/?path=/story/components-loading
```

**Review by francinelucca — COMMENTED** [review]
LGTM! 🎉

File: src/carbon.yml [hunk 10]
```diff
@@ -0,0 +1,426 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails/icon.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-icon
+    #TODO: @aubrey check tag
+    tags:
+      - contextual-navigation
```
Comment (aubrey-oneal) [inline src/carbon.yml:135]: Delete contextual-navigation tag. Add media and content-element tags.

File: src/carbon.yml [hunk 11]
```diff
@@ -0,0 +1,426 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails/input.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-input
+    #TODO: @aubrey check tag
+    tags:
+      - input-control
```
Comment (aubrey-oneal) [inline src/carbon.yml:159]: Approved 👍

File: src/carbon.yml [hunk 12]
```diff
@@ -0,0 +1,426 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails/radio.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-radio
+    #TODO: @aubrey check tag
+    tags:
+      - data-display
```
Comment (aubrey-oneal) [inline src/carbon.yml:269]: Approved 👍

File: src/carbon.yml [hunk 13]
```diff
@@ -0,0 +1,426 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails/context-menu.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-context-menu
+    #TODO: @aubrey check tag
+    tags:
+      - contextual-navigation
```
Comment (aubrey-oneal) [inline src/carbon.yml:85]: Delete contextual-navigation. Add input-control.

File: src/carbon.yml [hunk 14]
```diff
@@ -0,0 +1,426 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails/table.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-table
+    #TODO: @aubrey check tag
+    tags:
+      - data-display
```
Comment (aubrey-oneal) [inline src/carbon.yml:335]: Approved 👍

File: src/carbon.yml [hunk 15]
```diff
@@ -0,0 +1,426 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails/time-picker-select.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-time-picker-select
+    #TODO: @aubrey check tag
+    tags:
+      - data-display
```
Comment (aubrey-oneal) [inline src/carbon.yml:368]: Delete data-display. Add input-control.

File: src/carbon.yml [hunk 16]
```diff
@@ -0,0 +1,426 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails.tooltip-icon.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-tooltip-icon
+    #TODO: @aubrey check tag
+    tags:
+      - data-display
```
Comment (aubrey-oneal) [inline src/carbon.yml:408]: Approved 👍

File: src/carbon.yml [hunk 17]
```diff
@@ -0,0 +1,426 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails/skeleton.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-skeleton
+    #TODO: @aubrey check tag
+    tags:
+      - data-display
```
Comment (aubrey-oneal) [inline src/carbon.yml:302]: Delete data-display. Add system-feedback.

**Review by aubrey-oneal — CHANGES_REQUESTED** [review]
I approved several tagged components but suggested a few changes.

File: src/carbon.yml (hunk 13 above)
Comment (andreancardona) [inline src/carbon.yml:85]:
```suggestion
      - input-control
```

File: src/carbon.yml [hunk 18]
```diff
@@ -0,0 +1,426 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails/icon.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-icon
+    #TODO: @aubrey check tag
+    tags:
+      - contextual-navigation
```
Comment (andreancardona) [inline src/carbon.yml:135]:
```suggestion
      - media
      - content-element
```

File: src/carbon.yml [hunk 19]
```diff
@@ -0,0 +1,427 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails/list.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-list
+    #TODO: @aubrey check tag
+    tags:
+      - data-display
```
Comment (andreancardona) [inline src/carbon.yml:184]:
```suggestion
      - data-display
      - content-element
```

File: src/carbon.yml [hunk 20]
```diff
@@ -0,0 +1,428 @@
...
+    framework: angular
+    thumbnailPath: './thumbnails/time-picker-select.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-time-picker-select
+    #TODO: @aubrey check tag
+    tags:
+      - data-display
```
Comment (andreancardona) [inline src/carbon.yml:370]:
```suggestion
      - input-control
```

File: src/carbon.yml [hunk 21]
```diff
@@ -0,0 +1,419 @@
...
+    platform: web
+    framework: angular
+    thumbnailPath: './thumbnails/list.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://angular.carbondesignsystem.com/?path=/story/components-list
+    tags:
+      - data-display
```
Comment (andreancardona) [inline src/carbon.yml:180]:
```suggestion
      - data-display
      - content-element
```

File: src/carbon.yml [hunk 22]
```diff
@@ -0,0 +1,421 @@
...
+      - content-element
```
Comment (andreancardona) [inline src/carbon.yml:181]:
```suggestion
```

File: src/carbon.yml [hunk 23]
```diff
@@ -0,0 +1,420 @@
...
```
Comment (andreancardona) [inline src/carbon.yml:181]:
```suggestion
      - content-element
```

Comment (andreancardona) [inline src/carbon.yml:181]:
```suggestion
```

**Pull request merged**